README.md

```diff
@@ -16,7 +16,8 @@
 
 ## Live Demo 
 
-[liveDemoLink](https://elfin-git.github.io/Capstone-project-2/dist)
+[liveDemoLink](https://elfin-git.github.io/Capstone-project-2/dist/)
+You can click [here](https://drive.google.com/file/d/1vLHKtaT8Pp3Crx5S0LBR63rwnihQKqoE/view?usp=sharing) for the video presentation.
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you create a proper section for the **video** presentation to make your README more professional.
e.g.:
```
## Description Video

You can click [here](https://drive.google.com/file/d/1vLHKtaT8Pp3Crx5S0LBR63rwnihQKqoE/view?usp=sharing) for the video presentation.
```